tabbycat/results/forms.py

```diff
@@ -286,7 +286,7 @@ def get_preferences_options(self):
 
     @staticmethod
     def _fieldname_motion_veto(side):
-        return '%(side)s_motion_veto' % {'side': side}
+        return '%(side)d_motion_veto' % {'side': side}
```

Comment:
I'm guessing you tried it and these don't need to start with an alphabetic character. But I'd maybe prefix with `s` anyway so it's (slightly) less confusing to future developers?